### PR TITLE
Adding "Add condition" to breakpoints context menu

### PR DESCRIPTION
**Pull request by nyrosmith**
It was ways faster to add my changes on top of master than merging the existing ones. Especially all the debugger.properties changes were hard to track...
So closing https://github.com/devtools-html/debugger.html/pull/4094 in favor of this

### CODE REVIEW

**Comment by jasonLaster** [comment]
Looks good, on first pass

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -271,6 +271,8 @@ breakpointMenuItem.deleteAll2.label=Remove all
 breakpointMenuItem.deleteAll2.accesskey=a
 breakpointMenuItem.removeCondition2.label=Remove condition
 breakpointMenuItem.removeCondition2.accesskey=c
+breakpointMenuItem.addCondition2.label=Add condition
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Why addCondition2 and not just addCondition?

Comment (nyrosmith) [reply assets/panel/debugger.properties]: I didn't know why the others got renamed I I just wanted to ensure consistency. Should I remove the "2" from my new entries @flodolo

Comment (jasonLaster) [reply assets/panel/debugger.properties]: I'd keep the 2 to be consistent. We recently changed the labels to drop "breakpoint" e.g. `Remove breakpoint condition` to `Remove condition`...

**Pull request merged**